tools/topology/sof-hda-generic.m4

```diff
@@ -32,6 +32,11 @@ include(`platform/intel/intel-generic-dmic.m4')
 #
 # PCM0  <---> volume (pipe 1,2) <----> HDA Analog (HDA Analog playback/capture)
 # PCM1  <---> volume (pipe 3,4) <----> HDA Digital (HDA Digital playback/capture)
+
+include(`sof-hda-generic-idisp.m4')
```

Comment:
I don't think this makes sense either

Reply:
This trick is common used in a few topology files
https://github.com/thesofproject/sof/blob/66da4e89a9c2fce8fe04624a1001f7b8111a40c2/tools/topology/sof-cml-rt5682-max98357a.m4#L7-L9
https://github.com/thesofproject/sof/blob/66da4e89a9c2fce8fe04624a1001f7b8111a40c2/tools/topology/sof-cml-demux-rt5682-max98357a.m4#L7-L9
It is simple to keep the same part in different topology is the same.